app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -1921,6 +1928,11 @@ class BrowserTabViewModel @Inject constructor(
 
             evaluateSerpLogoState(url)
         }
+        viewModelScope.launch(dispatchers.io()) {
+            if (androidBrowserConfig.updateScriptOnPageFinished().isEnabled()) {
+                addDocumentStartJavaScript(webView)
```

Comment:
note: this was called from BrowserWebViewClient before, hence why you don't see any removals here